src/components/Migration.ts

```diff
@@ -67,3 +71,45 @@ async function removeV1(store : Store) : Promise<void> {
     })
 }
 migrations.push({ name: 'removeV1', func: removeV1 })
+
+enum CLIMigrationOption {
+    MIGRATE = 'Yes, migrate',
+    IGNORE = 'No, do not migrate',
+}
+
+async function migrateToCLIDataSource(store : Store) : Promise<void> {
+    const configuration = vscode.workspace.getConfiguration('vsflux')
+    if (configuration.get<string>('datasource') !== null) {
+        // Manually setting the value should skip this migration entirely.
+        return
+    }
+    const { stdout, stderr } = await exec('command -v influx')
+    if (stdout.length === 0 || stderr.length > 0) {
+        // We throw an error here because that's how we signal that we haven't applied
+        // a migration.
+        throw Error('Cannot migrate to cli config - cli not available')
+    }
+    const confirmation = await vscode.window.showInformationMessage(
+        'The influx command line tool was detected. Should the flux extension use that configuration for InfluxDB instances?',
```

Comment:
```suggestion
        'The influx command line tool was detected. Should the flux extension migrate the current InfluxDB configuration to the command line tool and use that configuration for InfluxDB instances?',
```

Or something, so it is clear that we also migrate the configuration, not just use whatever is currently in the CLI.

Reply:
This migration is disabled right now, because the more I think about it, the more I wonder if we should even bother migrating. As that string gets longer, the pop-up window gets a little harder to read, because it seems to be width-constrained. In fact, I removed a "Don't ask me this again" option and made things more direct specifically because the pop-up window was just too big.